### DISCUSSION TITLE
docs/self-hosted/publish-howto

### DISCUSSION THREAD

**Discussion by giscus** [flowershow/flowershow-app#45] [Announcements]
# docs/self-hosted/publish-howto

Learn how to publish your digital garden with Obsidian Flowershow plugin 🌷

https://flowershow.app/docs/self-hosted/publish-howto

**Comment by faekiva** [comment]
Images are broken here!

**Reply by rufuspollock** [reply]
Fixed. (A while ago). Thanks for flagging.

**Comment by juliuspopp** [comment]
Suggestion: GitLab support
Why? Access Control

For now I'll use Quartz because I want some form of free access control which I have with GitLab pages.

**Reply by rufuspollock** [reply]
@juliuspopp you can run Flowershow with Gitlab - there's nothing github specific.

**Reply by juliuspopp** [reply]
Ok, thanks for the clarification! 
so GitLab is just undocumented. 
As I am not that experienced with GitLab or GitHub and the static site generation process I am unsure whether I should expect some tweaking or whether it will just work.

**Comment by david-ortiz91** [comment]
I'm having an error message
Error: [next-mdx-remote] error compiling MDX:

**Reply by rufuspollock** [reply]
@david-ortiz91 can you provide a bit more detail of context - and maybe hop on the discord channel.

**Comment by Kelloliver** [comment]
nothing appear here just ´´Welcome to my flowershow site´´

**Comment by olayway** [comment]
I’m closing this issue since it concerns the old, now-deprecated self-hosted Flowershow Next.js template.
